packages/block-library/src/columns/index.js

```diff
@@ -24,8 +24,51 @@ export const settings = {
 		align: [ 'wide', 'full' ],
 		html: false,
 	},
+	example: {
+		innerBlocks: [
+			{
+				name: 'core/column',
+				innerBlocks: [
+					{
+						name: 'core/paragraph',
+						attributes: {
+							content: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent et eros eu felis pellentesque efficitur. Nam dapibus felis malesuada tincidunt rhoncus. Integer non malesuada tortor.',
+						},
+					},
+					{
+						name: 'core/image',
+						attributes: {
+							url: 'https://images.unsplash.com/photo-1549880339-d93e3072aef4',
+						},
+					},
+					{
+						name: 'core/paragraph',
+						attributes: {
+							content: 'Suspendisse commodo neque lacus, a dictum orci interdum et. Ut vel mi ut leo fringilla rutrum.',
+						},
+					},
+				],
+			},
+			{
+				name: 'core/column',
+				innerBlocks: [
+					{
+						name: 'core/paragraph',
+						attributes: {
+							content: __( 'Etiam et egestas lorem. Vivamus sagittis sit amet dolor quis lobortis. Integer sed fermentum arcu, id vulputate lacus. Etiam fermentum sem eu quam hendrerit, eget faucibus urna pulvinar.' ),
+						},
+					},
+					{
+						name: 'core/paragraph',
+						attributes: {
+							content: __( 'Nam risus massa, ullamcorper consectetur eros fermentum, porta aliquet ligula. Sed vel mauris nec enim ultricies commodo.' ),
+						},
+					},
+				],
+			},
+		],
+	},
```

Comment:
Needs overflow taken care of.

<img width="366" alt="Captura de ecrã 2019-09-23, às 14 19 05" src="https://user-images.githubusercontent.com/150562/65428915-29477200-de0d-11e9-984a-3bc97c8a7033.png">